src/pages/CategoriesPage/CategoriesPage.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styles from './CategoriesPage.module.scss';
+import Loader from 'components/Loader';
+import HorizontalCard from 'components/HorizontalCard';;
+import api from 'api';
+import { Category } from 'types';
+import { useNavigate } from "react-router-dom";
+
+const CategoriesPage: React.FC = () => {
+    const [isLoading, setLoading] = React.useState<boolean>(true);
```

Comment:
здесь тип очевидный и TS выводит сам:
```suggestion
    const [isLoading, setLoading] = React.useState(true);
```